src/_data/authors.yml

```diff
@@ -0,0 +1,128 @@
+anho:
+    name: Andrei Hogea
+    occupation: iOS Developer
+    github: kjoneandrei
+    about: iOS developer based in Copenhagen, Denmark. Always curious
+    image: https://media.licdn.com/dms/image/C4E03AQGAyZLJCiZ2qg/profile-displayphoto-shrink_200_200/0?e=1526220000&v=alpha&t=QqEvuqH3E93VlTXOs5VGTDJW8eetHOUzczgEqsqA6Ao
+brto:
+    name: Brett R. Toomey
+    occupation: Vapor & iOS Developer
+    about: Using Swift to take over the world.
+    twitter: brettRToomey
+    github: brettRToomey
+    image: https://pbs.twimg.com/profile_images/803734242597314560/aeWxuf3k_400x400.jpg
+chco:
+    name: Chris Combs
+    occupation: Lead iOS Developer
+    twitter: chriscombs76
+    github: chriscombs
+    about: iOS Developer, Swift enthusiast
+    image: https://avatars2.githubusercontent.com/u/3409077?v=4&u=6aef567e861729b4448c3f70f5cf6a0de6fe7768&s=400
+dhni:
+    name: Dennis Haulund Nielsen
+    occupation: Front-end Developer
+    about: I love Javascripts
+    github: nickskull
+    image: https://lh4.googleusercontent.com/-8zlAxSM5xCQ/AAAAAAAAAAI/AAAAAAAAABE/t7_zkJLB8_0/photo.jpg
+doha:
+    name: Dominik Hádl
+    occupation: Lead iOS Developer
+    twitter: dominikhadl
+    github: nickskull
+    about: iOS games and apps developer, music enthusiast, game designer and stuff. I love mechanical keyboards. /ˈɡɪf/
+    image: https://pbs.twimg.com/profile_images/736536321867448320/4OmC9TeR.jpg
+dori:
+    name: Dominik Ringler
+    occupation: iOS Developer
+    twitter: dominikringler
+    github: crashoverride777
+    about: iOS games and apps developer. I love Swift very much. /ˈɡɪf/
+    image:
```

Comment:
Let's see if we can find an image for everyone. We should also have some avatar fall back image - maybe the designers can help out with this.

Reply:
Maybe designers can prepare or can we use images from old posts?